services/provider/server/server.go

```diff
@@ -969,6 +986,223 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 
 }
 
+// SHIM for GetStorageClaimConfig to return response to 4.18 clients
+func (s *OCSProviderServer) proxyGetStorageClaimConfigBlock(
+	req *pb.StorageClaimConfigRequest,
+	storageCluster *ocsv1.StorageCluster,
+	consumerConfig util.StorageConsumerResources,
```

Comment:
Don't send the wrapper between function, sand the raw map/config map and wrap it only for local use